docs/docsite/rst/dev_guide/testing/sanity/no-underscore-variable.rst

```diff
@@ -0,0 +1,28 @@
+Sanity Tests » no-underscore-variable
+=====================================
+
+In the future, Ansible may use the identifier ``_`` to internationalize its
+message strings.  To be ready for that, we need to make sure that there are
+no conflicting identifiers defined in the code base.
+
+In common practice, ``_`` is frequently used as a dummy variable (a variable
+to receive a value from a function where the value is useless and never used).
+In Ansible, we're using the identifier ``dummy`` for this purpose instead.
```

Comment:
Given that last test, it doesn't look like information about this really belongs in no-underscore-variable.rst .  If someone is affected by this, they would already have hit it with ```_```.